### ISSUE TITLE
Support for multiple Redis Sentinels

### ISSUE THREAD

**Issue by taai** [phpredis/phpredis#2132]
My knowledge about the Redis Sentinel is very limited right now, so, this is probably going to be just a question about how to use this package properly...

I have 3 Sentinel nodes (as there should be by recommendations), but in the [README](https://github.com/phpredis/phpredis/blob/5.3.7/sentinel.markdown) I see only how to use a single Sentinel node... The ```RedisSentinel()``` class accepts only single host (IP address or hostname).

Am I supposed to manage the connections to Sentiinel nodes myself? For example, should I connect to first Sentinel node (that is down) with class ```RedisSentinel()```, catch an exception and try the next Sentinel node until no Sentinel nodes are left to try? At least. I think that [_Predis_ does exactly that](https://github.com/predis/predis/blob/v2.0.0/src/Connection/Replication/SentinelReplication.php#L299).

Are there any drawbacks why multiple multiple Sentinels are not (yet) supported (the management is not built-in)?

### Expected behaviour

Should be able to use multiple sentinels and use another sentinel when one is down.

### Actual behaviour

Only single host can be used.

### I'm seeing this behaviour on
- OS: Ubuntu 22.04
- Redis: 6.2
- PHP: 8.1
- phpredis: 5.3.7

### Steps to reproduce, backtrace or example script

Just fallow the instructions in dedicated [README](https://github.com/phpredis/phpredis/blob/5.3.7/sentinel.markdown).

### I've checked
- [x] There is no similar issue from other users
- [x] Issue isn't fixed in `develop` branch

**Comment by yatsukhnenko** [comment]
> Am I supposed to manage the connections to Sentiinel nodes myself?

Yes, current implementation doesn't imply multiple sentinel nodes so you have to manage them by yourself

**Comment by sartor** [comment]
It is very strange, that high availability service does not support multiple nodes. This is the only reason i still using predis instead of this native one
